spec/ruby/core/string/scrub_spec.rb

```diff
@@ -105,4 +105,25 @@
     input.scrub! { |b| "<?>" }
     input.should == "a<?>"
   end
+
+  it "does not maintain the state of tainted strings that are already valid" do
```

Comment:
That description is confusing, I believe what happens is that `taint` does nothing on Ruby 2.7+.
I'd recommend just removing this spec, `taint` is deprecated/noop anyway in 2.7+.

Reply:
good idea; thanks!